tests/cypress/e2e/api/graphqlPermissions.cy.ts

```diff
@@ -0,0 +1,182 @@
+import gql from 'graphql-tag';
+import {
+    addNode,
+    createUser,
+    deleteUser,
+    grantRoles,
+    revokeRoles,
+    setNodeProperty,
+    addUserToGroup
+} from '@jahia/cypress';
+
+describe('Test graphql permissions', () => {
+    let subList2Cursor: string;
+
+    before('create user and test data', () => {
+        // DeleteNode('/testList'); // TODO should not be needed
+        createUser('testUser', 'testPassword');
+        addUserToGroup('testUser', 'privileged');
+
+        /*
+        Test data to create:
+        /testList (jnt:contentList) - owner: true
+            /testSubList1 (jnt:contentList) - owner: false
+            /testSubList2 (jnt:contentList) - owner: true
+            /reference1 (jnt:contentReference) - owner: false
+                [j:node=ref->/testSubList2]
+            /reference2 (jnt:contentReference) - owner: true
+                [j:node=ref->/testSubList2]
+         */
+
+        addNode({
+            parentPathOrId: '/',
+            name: 'testList',
+            primaryNodeType: 'jnt:contentList',
+            children: [
+                {
+                    name: 'testSubList1',
+                    primaryNodeType: 'jnt:contentList'
+                },
+                {
+                    name: 'testSubList2',
+                    primaryNodeType: 'jnt:contentList'
+                },
+                {
+                    name: 'reference1',
+                    primaryNodeType: 'jnt:contentReference'
+                },
+                {
+                    name: 'reference2',
+                    primaryNodeType: 'jnt:contentReference'
+                }
+            ]
+        })
+            .then(response => {
+                // Keep reference to testSubList2
+                subList2Cursor = response.data.jcr.addNode.addChildrenBatch[1].uuid;
+                setNodeProperty('/testList/reference1', 'j:node', subList2Cursor, 'en');
+                setNodeProperty('/testList/reference2', 'j:node', subList2Cursor, 'en');
+            })
+            .then(() => {
+                // Tweak the permissions
+                grantRoles('/testList', ['owner'], 'testUser', 'USER');
+                grantRoles('/testList', ['privileged'], 'testUser', 'USER');
+                revokeRoles('/testList/testSubList1', ['owner'], 'testUser', 'USER');
+                revokeRoles('/testList/reference1', ['owner'], 'testUser', 'USER');
+            }
+            );
+    });
+
+    after('Delete user and test data', () => {
+        deleteUser('testUser');
+        cy.apolloClient() // TODO needed but I don't know why
```

Comment:
I can't figure out why the credentials have to be reset (by default when creating an `apolloClient` with no parameters).
Without it, the temp credentials used in the rest of the tests are used (testUser/testPassword)

Reply:
Remove the Todo from the comment. Only the explanation as a comment is excepted  ; there is nothing to do now.
// reset apollo client authentication credentials to avoid that following tests use the ones in that one.